tools/libvirt_test.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -euo pipefail
 
+#
+# tests that guest images are buildable through the weldr API and boot-tests them using libvirt
```

Comment:
I'd say either "using the weldr API" or just "using composer-cli".. Idk if it's immediately obvious what "boot-tests them" is so I'd just say "and verifies they boot with cloud-init using libvirt"